File: Python_Programming/numbers/display_cube.py
def DisplayCube(No):

    print("Cube of given number is : ",(No*No*No))

def main():

    iValue = 0

    print("Enter the number : ",end="")
    iValue = int(input())

    DisplayCube(iValue)
    
if __name__ == "__main__":
    main()

File: Cpp_programming/numbers/display_cube.cpp
// write program which accepts one number and prints cube of that number 

#include<iostream>
using namespace std;

void DisplayCube(int iNo)
{
    cout<<"Cube of given number is : "<<(iNo * iNo * iNo)<<"\n";
}

int main()
{
    int iValue = 0;

    cout<<"Enter the number : \n";
    cin>>iValue;

    DisplayCube(iValue);
    
    return 0;
}